.github/ISSUE_TEMPLATE/convergence_epic.md

```diff
@@ -1,8 +1,8 @@
 ---
 name: (internal) Component implementation epic
 about: (team member use only) Epic issue tracking implementation of a particular component
----
 
+---
```

Comment:
This change doesn't seem to have an effect. What is it fixing, and is it necessary?

Reply:
this is breaking frontmatter as well